tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/ExecuteCommandTest.php

```diff
@@ -0,0 +1,136 @@
+namespace Doctrine\DBAL\Migrations\Tests\Tools\Console\Command;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Doctrine\DBAL\Migrations\Version;
+use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use Doctrine\DBAL\Migrations\Tests\MigrationTestCase;
+use Doctrine\DBAL\Migrations\Tools\Console\Command\ExecuteCommand;
+
+class ExecuteCommandTest extends MigrationTestCase
+{
+    const VERSION = '20160705000000';
+
+    private $commmand, $app, $config, $version, $questions;
+
+    public function testWriteSqlCommandOutputsSqlFileToTheCurrentWorkingDirectory()
+    {
+        $this->version->expects($this->once())
+            ->method('writeSqlFile')
+            ->with(getcwd(), 'up');
+
+        $tester = $this->executeCommand([
+            '--write-sql' => true,
+            '--up' => true,
+        ]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    public function testWriteSqlOutputsSqlFileToTheSpecifiedDirectory()
+    {
+        $this->version->expects($this->once())
+            ->method('writeSqlFile')
+            ->with(__DIR__, 'down');
+
+        $tester = $this->executeCommand([
+            '--write-sql' => __DIR__,
+            '--down' => true,
+        ]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    public function testNoMigrationIsExecuteWhenTheUserDoesNotConfirmTheAction()
+    {
+        $this->questions->expects($this->once())
+            ->method('ask')
+            ->willReturn(false);
+        $this->version->expects($this->never())
+            ->method('execute');
+
+        $tester = $this->executeCommand([]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    public function testMigrationsIsExecutedWhenTheUserConfirmsTheAction()
+    {
+        $this->questions->expects($this->once())
+            ->method('ask')
+            ->willReturn(true);
+        $this->version->expects($this->once())
+            ->method('execute')
+            ->with('up', true, true);
+
+        $tester = $this->executeCommand([
+            '--dry-run' => true,
+            '--query-time' => true,
+        ]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    public function testMigrationIsExecutedWhenTheConsoleIsNotInInteractiveMode()
+    {
+        $this->questions->expects($this->never())
+            ->method('ask');
+        $this->version->expects($this->once())
+            ->method('execute')
+            ->with('up', true, true);
+
+        $tester = $this->executeCommand([
+            '--dry-run' => true,
+            '--query-time' => true,
+        ], ['interactive' => false]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    protected function setUp()
+    {
+        $this->questions = $this->getMock(QuestionHelper::class);
```

Comment:
You just need to add something similar to the test here https://github.com/doctrine/migrations/blob/master/bin/doctrine-migrations.php#L74-L78 and then the last test will pass.

Thanks a lot for the PR.